doc/changelog/02-specification-language/15789-coercions-no-uniform-cond.rst

```diff
@@ -0,0 +1,12 @@
+- **Removed:**
+  the uniform inheritance condition for coercions,
+  allowing more freedom on parameters of coercion that are now inferred
+  using unification or even canonical structures or typeclasses
```

Comment:
```suggestion
- **Removed:**
  the uniform inheritance condition for coercions,
  allowing more freedom for the parameters of coercions that are now inferred
  using unification or even canonical structures or typeclasses
```

It's odd to refer to this as "removed" when IIUC you're improving functionality by eliminating a previous limitation--an enhancement.  There is no compatibility issue IIUC.


"using unification or even canonical structures or typeclasses" - unclear. Should this be more like "using unification, canonical structures or typeclasses"?  Or "using unification on canonical structures or typeclasses"?  "even" seems unnecessary.

Reply:
Indeed, I moved it to `**Added:**`